addons/mail/static/src/discuss/call/web/discuss_patch.js

```diff
@@ -1,11 +1,28 @@
 /* @odoo-module */
 
 import { Discuss } from "@mail/core/common/discuss";
+import { useEffect } from "@odoo/owl";
 
 import { patch } from "@web/core/utils/patch";
 
 patch(Discuss.prototype, "discuss/call/web", {
+    setup(...args) {
+        this._super(...args);
+        Object.assign(this.state, { openInviteButton: 0 });
```

Comment:
why not this.state.openInviteButton = 0?

Reply:
`Object.assign()` benefits are that shape for definition is similar to defining object in `useState()`, and it prevents repeating `this.state.X =` if we add more things to the state.

But that's very minor, so I'm somewhat indifferent with either `Object.assign()` or `this.state.X = `.